components/contact/ContactField.tsx

```diff
@@ -0,0 +1,42 @@
+import { ContactFieldType } from '@/types';
+
+const ContactField = ({
+  label,
+  id,
+  type = 'text',
+  component = 'input',
+  getInputStyle,
+  register,
+  errors,
+}: ContactFieldType) => {
+  const inputProps = {
+    ...register(id),
+    type,
+    className: getInputStyle(errors[id]),
+  };
+
+  return (
+    <div className="max-w-[21.6rem] s:max-w-[35rem] md:max-w-[40rem]">
+      <label
+        htmlFor={id}
+        className="mb-4 block text-lg font-normal leading-[160%] text-black-300 dark:text-white-900 md:text-xl"
+      >
+        {label}
+      </label>
+
+      {component === 'input' ? (
+        <input {...inputProps} />
+      ) : component === 'textarea' ? (
+        <textarea {...inputProps}></textarea>
```

Comment:
Consider adding `resize-none` as a className here. That way you can disable the resizing and match the design.

Reply:
@AlexDjangoX, I just saw this in another PR Review. You mustn't work on the same thing in multiple files. Not only will it cause merge conflicts, but it will have two mentors reviewing/addressing the same issue. Thanks!